packages/terra-form-select/src/search/Frame.jsx

```diff
@@ -199,14 +216,30 @@ class Frame extends React.Component {
     );
   }
 
+  /**
+   * TODO: docme
+   */
+  handleMenuTabDown(event) {
+    const { current: selectInput } = this.selectInputRef;
+
+    if (selectInput) {
+      selectInput.focus();
+      this.handleKeyDown(event);
+    }
+  }
+
   /**
    * Closes the dropdown.
    */
   closeDropdown() {
+    const { current: selectInput } = this.selectInputRef;
+    const { current: select } = this.selectRef;
+
     this.setState({
       isAbove: false,
-      isFocused: document.activeElement === this.input || document.activeElement === this.select,
+      isFocused: document.activeElement === selectInput || document.activeElement === select, // TODO: fixme
```

Comment:
What needs fixed here?

Reply:
This is tied to focus management/blur.. working through the logic to determine when the select is focused is complicated since `document.activeElement` seems to consistently be `body` when the menu is closed

debugging this right now